composer.json

```diff
@@ -60,7 +60,7 @@
         "doctrine/doctrine-cache-bundle": "~1.0",
         "mmoreram/simple-doctrine-mapping": "~0.1",
 
-        "sebastian/money": "~1.3",
+        "sebastian/money": "1.5.*",
```

Comment:
I'm talking about this update only add new methods. Not change any previous methods used by Elcodi and add some news like getConvertedAmount, that is very useful when you work with API's and Twig extension is not valid to format php output.